test/API/test_calls.py

```diff
@@ -51,16 +51,24 @@ def test_get_called_with(self, get_request, create_session):
         library = APILibrary()
         library.call_get_request({"a": "Text"}, "Endpoint", "fullstring")
         get_request.assert_called_with("getapi", "fullstring")
-        create_session.assert_called_with("getapi", "Endpoint", {"a": "Text"})
+        create_session.assert_called_with("getapi", "Endpoint", {"a": "Text"}, cookies=None)
 
     @patch('RequestsLibrary.RequestsKeywords.create_session')
     @patch('RequestsLibrary.RequestsKeywords.get_request')
     @patch('requests.packages.urllib3.disable_warnings')
-    def test_get_insecure_request(self, create_session, get_request, disable_warnings):
+    def test_get_insecure_request(self, get_request, create_session, disable_warnings):
```

Comment:
this should be  def test_get_insecure_request(self, disable_warnings, get_request, create_session):

If you do a negative test assert_not_called, for disable warnings it would say that create_session was called once